.github/workflows/keep_docker_in_sync.yml

```diff
@@ -0,0 +1,59 @@
+name: Keep Docker in Sync
+
+on:
+  schedule:
+    - cron: '0 7 * * *'  # Every day at 07:00 UTC (1AM CST or 2AM CDT)
+
+jobs:
+  keep-docker-in-sync:
+    name: Check for Latest Version
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Setting up Python
+        uses: actions/setup-python@v1.1.1
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Find latest pmdarima version
+        id: pypi_version_check  # Need this to refer to output below
+        run: |
+          pip install yolk3k
```

Comment:
What is this package?

Reply:
I just googled `find most recent version of PyPI package` and it came up ([Homepage](https://github.com/myint/yolk)). It's a fork of a Python 2 package. Both seem to be rarely updated 😕  But it just returns this, so it works for our purposes

```
pmdarima 1.6.1
```

Reply:
Would you rather I use the info at https://pypi.python.org/pypi/pmdarima/json and parse it myself?

Reply:
Because this works as well:

```python
import json

import requests

pmdarima = json.loads(requests.get('https://pypi.python.org/pypi/pmdarima/json').text)
versions = [version for version in pmdarima['releases'].keys()]
print(max(versions))
```

Reply:
^ I tried this, and it didn't work in #14, so I am just going to leave this